src/main/java/com/wcc/platform/service/PlatformService.java

```diff
@@ -42,12 +46,24 @@ public ResourceContent saveResourceContent(final ResourceContent resourceContent
 
   /** Save any type of page based on page Type. */
   public Object savePage(final LandingPage page) {
-    return lpRepository.save(page);
+    try {
+      return pageRepository.save(objectMapper.writeValueAsString(page));
+    } catch (JsonProcessingException e) {
+      throw new PlatformInternalException(PageType.LANDING_PAGE, e);
+    }
   }
 
   /** Save any type of page based on page Type. */
   public Object savePage(final FooterPage page) {
-    return footerRepository.save(page);
+    try {
+      return pageRepository.save(objectMapper.writeValueAsString(page));
```

Comment:
TODO: Investigate why it is not saving =/

Reply:
Does the saving work for the landing page above?